openwisp_firmware_upgrader/hardware.py

```diff
@@ -16,6 +16,10 @@
 OPENWRT_FIRMWARE_IMAGE_MAP.update(
     OrderedDict(
         (
+            (
+                'ramips-mt76x8-gl-mt300n-v2-squashfs-sysupgrade.bin',
+                {'label': 'GL.iNet GL-MT300N-V2', 'boards': ('GL-MT300N-V2',)},
+            ),
```

Comment:
when you need to add this kind of stuff, you can just push directly to master, you should have the required permissions if I'm not mistaken